<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OThruster</class>
 <widget class="QWidget" name="OThruster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>97</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QScrollBar" name="PitchThrust">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>18</width>
     <height>131</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="PitchT">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollBar" name="YawThrust">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>20</width>
     <height>120</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="RollThrust">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>20</width>
     <height>120</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="RollT">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>168</y>
     <width>51</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Range</string>
   </property>
   <widget class="QRadioButton" name="YTenth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>.1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="YOne">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="YTen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>51</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Range</string>
   </property>
   <widget class="QRadioButton" name="PTen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="POne">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="PTenth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>.1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>51</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Range</string>
   </property>
   <widget class="QRadioButton" name="RTenth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>.1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="RTen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ROne">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="YawT">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PitchThrust</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>OThruster</receiver>
   <slot>SetPitchValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YawThrust</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>OThruster</receiver>
   <slot>SetYawValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RollThrust</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>OThruster</receiver>
   <slot>SetRollValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YawThrust</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>OThruster</receiver>
   <slot>SetYawValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>POne</sender>
   <signal>toggled(bool)</signal>
   <receiver>OThruster</receiver>
   <slot>PitchRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PTen</sender>
   <signal>toggled(bool)</signal>
   <receiver>OThruster</receiver>
   <slot>PitchRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PTenth</sender>
   <signal>toggled(bool)</signal>
   <receiver>OThruster</receiver>
   <slot>PitchRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YOne</sender>
   <signal>toggled(bool)</signal>
   <receiver>OThruster</receiver>
   <slot>YawRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YTen</sender>
   <signal>toggled(bool)</signal>
   <receiver>OThruster</receiver>
   <slot>YawRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YTenth</sender>
   <signal>toggled(bool)</signal>
   <receiver>OThruster</receiver>
   <slot>YawRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ROne</sender>
   <signal>toggled(bool)</signal>
   <receiver>OThruster</receiver>
   <slot>RollRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RTen</sender>
   <signal>toggled(bool)</signal>
   <receiver>OThruster</receiver>
   <slot>RollRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RTenth</sender>
   <signal>toggled(bool)</signal>
   <receiver>OThruster</receiver>
   <slot>RollRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PitchThrust</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>OThruster</receiver>
   <slot>SetPitchValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>9</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RollThrust</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>OThruster</receiver>
   <slot>SetRollValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>9</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Exit()</slot>
  <slot>SetPitchValue()</slot>
  <slot>SetYawValue()</slot>
  <slot>SetRollValue()</slot>
  <slot>PitchRange()</slot>
  <slot>YawRange()</slot>
  <slot>RollRange()</slot>
 </slots>
</ui>
